staging/src/k8s.io/kubectl/pkg/cmd/get/sorter.go

```diff
@@ -406,15 +430,17 @@ func NewTableSorter(table *metav1.Table, field string) (*TableSorter, error) {
 		}
 	}
 
-	if len(table.Rows) > 0 && !fieldFoundOnce {
-		return nil, fmt.Errorf("couldn't find any field with path %q in the list of objects", field)
```

Comment:
So I guess the original intention was to fail if the field was not found in the list of objects.

I wonder if anyone (other than kubectl) is depending on this behavior.

I'm wondering if we should preserve the behavior of NewTableSorter, as-is, and handle the error properly in kubectl.

If this error is returned, why isn't it being printed anywhere? :thinking: 

It looks like the problem is here:
https://github.com/kubernetes/kubernetes/blob/a86380c7813a6d0cfa248c9165c878038730526a/staging/src/k8s.io/kubectl/pkg/cmd/get/get.go#L559

It's not checking if `PrintObj` returns an error.

Reply:
@brianpursley Nice find! Do you think that is what the behavior should be? (The bug report ticket said it should fall back to the behavior where no filter was applied.)